components/murata/NCP03WF104F05RL.stanza

```diff
@@ -0,0 +1,55 @@
+#use-added-syntax(jitx)
+defpackage ocdb/components/murata/NCP03WF104F05RL :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/utils/defaults
+
+  import ocdb/utils/landpatterns
+  import ocdb/utils/symbols
+  import ocdb/utils/box-symbol
+  import ocdb/utils/bundles
+  import ocdb/utils/generator-utils
+  import ocdb/utils/generic-components
+  import ocdb/utils/tolerance
+  import ocdb/utils/property-structs
+
+public pcb-component component : 
+  manufacturer = "Murata"
+  description = "THERM NTC 100KOHM 4250K 0201"
+  mpn = "NCP03WF104F05RL"
+
+  port p : pin[[1 2]]
+
+  val sym = resistor-sym(ResistorTherm)
+  symbol = sym(p[1] => sym.p[1], p[2] => sym.p[2])
+  val pkg = ipc-two-pin-landpattern("0201")
+  landpattern = pkg(p[1] => pkg.p[1], p[2] => pkg.p[2])
+
+  val generic-props = GenericPin(min-max(0.0, 5.0), 2000.0)
+
+  property(p[1].generic-pin) = generic-props
```

Comment:
generic-pin was meant more for IC pins. I would replace this with a property associated with resistors, like 

```
rated-voltage = 5.0
```

Reply:
Ok, that makes sense. I just looked for any "compatible" property that had a max-voltage associated with it.

Reply:
BTW, the PWL macro only takes tuples of doubles, but we may want to upgrade it to accept a Tuple<Double, Toleranced> -> Toleranced? (another PR, I think)

Reply:
Technically the `rated-voltage` property should be a relative voltage (i.e. the max voltage difference between p[1] and p[2] for rated-voltage = 5.0. But I am not sure how to code that (yet).